### ISSUE TITLE
Freeglut 3.x : glutSolidSphere and glutSolidTorus Error

### ISSUE THREAD

**Issue by gerard-durand** [FreeGLUTProject/freeglut#126]
Some time ago, I have opened a bug report on https://sourceforge.net/p/freeglut/bugs/254/
I don't see it as solved or a copy on this github repository.
It seems that this bug is always here in freeglut 3.2.2

**Comment by jtsiomb** [comment]
Yes, we didn't move the bug reports, the sourceforge bug tracker is still active, this one is just the preferred one now.
